manifests/FTB/FTB/1.0.8.yaml

```diff
@@ -0,0 +1,15 @@
+Id: FTB.FTB
+Version: 1.0.8
+Name: FTBApp
+Publisher: Feed The Beast
+License: LGPL 2.1
```

Comment:
While this is true for the frontend, its not true for the backend (https://github.com/CreeperHost/modpacklauncher) which the installer is built from.

Reply:
Fair point, I updated it to mention the licenses for both the frontend and the backend